tic_tac_toe/src/board_test.py

```diff
@@ -0,0 +1,229 @@
+from board import Board
+
+
+def update_column_1():
```

Comment:
These helper function names should be descriptive of what situations they're testing.  I know that you're updating columns and such, but I don't know what situation you're testing.

Also, and more generally, are you reusing any of these functions?  If not, perhaps you should just set these inside of the tests.